docroot/modules/custom/govcms_draft_content_access/js/govcms_dca.js

```diff
@@ -0,0 +1,15 @@
+(function ($) {
```

Comment:
We should pass in the Drupal object here as well so we're not implicitly using the global object.

``` js
(function (Drupal, $) {
})(Drupal, jQuery);
```